halconfig/settings.js

```diff
@@ -10,6 +10,7 @@ var fiatEnabled = '{%features.fiat%}' === 'true';
 var jobsEnabled = '{%features.jobs%}' === 'true';
 var pipelineTemplatesEnabled = '{%features.pipelineTemplates%}' === 'true';
 var artifactsEnabled = '{%features.artifacts%}' === 'true';
+var canaryEnabled = '{%features.canary%}' === 'true';
```

Comment:
Not sure what you're planning on naming the corresponding Halyard flags, but I'd like to refer to that system as `legacyCanary` or something. In the near future there will be a fully open-source canary system for Spinnaker, and I'd like to avoid calling that one `canaryV2`.

Reply:
Will update feature name exposed by hal as legacyCanary. This is to get halconfig in sync with feature present under deck/settings.js. Feature name under feature section will remain "canary" as that is the name under which the feature is implemented in deck. Is that acceptable?

Reply:
Yes, that's totally fine. Thanks.